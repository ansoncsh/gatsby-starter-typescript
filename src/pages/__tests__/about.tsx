import * as React from "react"
import * as renderer from "react-test-renderer"

import AboutPage from "../about"

describe("AboutPage", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<AboutPage />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
