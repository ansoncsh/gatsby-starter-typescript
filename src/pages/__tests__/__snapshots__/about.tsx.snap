// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage renders correctly 1`] = `
<div>
  <p>
    Welcome to about page
  </p>
  <a
    aria-current="page"
    className=""
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={Object {}}
  >
    Go back to the homepage
  </a>
</div>
`;
